### PR TITLE
Draft: Make namespace UsersUtils public

### PR DESCRIPTION
**Pull request by viktor-liablin**
Make namespace UsersUtils public

### CODE REVIEW

File: src/users/index.js [hunk 1]
```diff
@@ -14,11 +14,15 @@ export default class Users {
     this.dataStore = this.app.Data.of(User)
   }
 
+  Utils() {
```
Comment (Valodya) [inline src/users/index.js:17]: are you sure it should be a method?

File: backendless.d.ts [hunk 2]
```diff
@@ -276,6 +276,14 @@ declare module Backendless {
 
         let currentUser: Backendless.User;
 
+        /**
+         * @public
+         * @namespace Backendless.UserService.Utils
+         */
+        namespace Utils {
+            function getClientUserLocale(): string;
```
Comment (Valodya) [inline backendless.d.ts:284]: need to add `tsd` tests

File: src/users/index.js [hunk 3]
```diff
@@ -14,11 +14,15 @@ export default class Users {
...
+    return UsersUtils
```
Comment (Valodya) [inline src/users/index.js:18]: cover it with a unit test

File: backendless.d.ts (hunk 2 above)
Comment (viktor-liablin) [reply backendless.d.ts:284]: +

File: src/users/index.js (hunk 1 above)
Comment (viktor-liablin) [reply src/users/index.js]: fixed

File: src/users/index.js (hunk 3 above)
Comment (viktor-liablin) [reply src/users/index.js:18]: \+

**Pull request merged**